cachito/web/migrations/versions/4d17dec0cfc3_adjust_unique_constraint.py

```diff
@@ -0,0 +1,38 @@
+"""Adjust unique constraint on Flag table
+Remove unique constraint from id and name but keep name unique
+
+Revision ID: 4d17dec0cfc3
+Revises: c6ac095d8e9f
+Create Date: 2021-05-25 13:21:48.298960
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "4d17dec0cfc3"
+down_revision = "c6ac095d8e9f"
+branch_labels = None
+depends_on = None
+
+
+flag_table = sa.Table(
+    "flag",
+    sa.MetaData(),
+    sa.Column("id", sa.Integer(), primary_key=True),
+    sa.Column("name", sa.String(), nullable=False),
+    sa.Column("active", sa.Boolean(), nullable=False, default=True),
+)
+
+
+def upgrade():
+    with op.batch_alter_table("flag") as batch_op:
+        batch_op.create_index(batch_op.f("ix_flag_name"), ["name"], unique=True)
```

Comment:
Could this be `create_unique_constraint` instead? If yes, I would also rename it to `flag_name_key`.